src/ride/ride.c

```diff
@@ -1188,11 +1188,14 @@ int sub_6C683D(int* x, int* y, int* z, int direction, int type, uint16 extra_par
 			mapElement->properties.track.colour &= 0x0F;
 			mapElement->properties.track.colour |= (extra_params & 0xFF) << 4;
 		}
+		
 		if (flags & (1 << 3)) {
+			// Cable lift hill track
 			mapElement->properties.track.colour |= (1 << 3);
 		}
 		if (flags & (1 << 4)) {
-			mapElement->properties.track.colour &= 0xF7;
+			// Clear Cable lift hill track
+			mapElement->properties.track.colour &= ~(1 << 3);
```

Comment:
Like I have done with lift hill, make the functions is_track_cable_lift_hill(trackElement) and set_track_cable_lift_hill(bool).